data/sql/updates/pending_db_world/rev_1615103299173233870.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1615103299173233870');
+
+-- Kolkar Drudge and Kolkar Outrunner spell resistances
+
```

Comment:
Redundant blank line, or is there something I don't know about AC codestyle?
```suggestion
```

Reply:
The content looks good otherwise. Looking forward to see the SQL added to the DB.

Reply:
Yeah, my mistake!